package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "string-length",
-	"version": "3.1.0",
+	"version": "4.0.0",
```

Comment:
You should not bump the version in a PR. It's up to the maintainer to decide what version and when to do that.

Reply:
Fixed.